README.md

```diff
@@ -58,6 +58,7 @@ Name                                                      | Status       | Short
 [Parallel-rustc](working-groups/parallel-rustc/)          | Active       | Making parallel compilation the default for rustc                                                  | [#t-compiler/wg-parallel-rustc][parallel-rustc_stream]
 [Profile-Guided Optimization](working-groups/pgo/)        | Active       | Implementing profile-guided optimization for rustc                                                 | [#t-compiler/wg-profile-guided-optimization][pgo_stream]
 [MIR Optimizations](working-groups/mir-opt/)              | Active       | Write MIR optimizations and refactor the MIR to be more optimizable.                               | [#t-compiler/wg-mir-opt][mir-opt-stream]
+[Rustc pipelining](working-groups/pipelining/)            | Active       | Enable Cargo to invoke rustc in a pipelined fashion, speeding up crate graph compiles.             | [#t-compiler/wg-pipelining][pipelining-stream]
```

Comment:
Can you also insert a entry to the [check-in table](https://github.com/rust-lang/compiler-team/blob/master/about/triage-meeting.md#working-group-check-in) after mir-opt? It occurs to me the "create a working group" procedure is missing this step.

Reply:
Also, [rust-lang/team](https://github.com/rust-lang/team) should be updated.